code/go/0chain.net/smartcontract/zcnsc/handler.go

```diff
@@ -57,10 +56,10 @@ func (zrh *ZcnRestHandler) getAuthorizerNodes(w http.ResponseWriter, r *http.Req
 		return
 	}
 
-	var (
-		authorizers []event.Authorizer
-		err         error
-	)
+	var err error
+
+	authorizers := make([]event.Authorizer, 0)
```

Comment:
how should help? you are guaranteed to receive error is no records are found

Reply:
Please, remember, that it's a public API. Approach of returning error if no records found is acceptable in ORMs, but not public APIs.

Reply:
I didn't get you, check the code, in line 66 and 64 you initialize this slice or process the error, you simply do nothing with this initialization.

Reply:
That initialization is needed, because of the logic of default JSON encoder. If your slice is empty and not initialized with "make" it will be interpreted as "null" in JSON.

Reply:
@dabasov this will create an instance of authorizer slice that has empty items, so that http response will return 200 with empty list, while if we return nil, the common.Response will return `StatusNoContent`,  which I think the client ends does not work with it.